DeAceroDLabs/marketplace-frontend
# [DMP-692] Added Offers route

## What is done

[[DMP-692] Vista de Paquetes](https://deacero.atlassian.net/browse/DMP-692)

## How is done

1. Added a new view for Offers.
2. Added the new view as a valid route on ShoppingCart.
3. Added a new step in the shoppingcart stepper.

## How to test

1. Run the project via `npm start`.
2. Go to `/carrito/ofertas`. 
3. If you have products in your shopping cart, you should see an empty screen with the shopping cart icon selected.
4. If you don't have any product in your shopping cart, add a new one.
  - Go home.
  - Click on the first product (Clavo 2" - 5").
  - Select Reforzado -> 51 -> 11 -> caja 25kg
  - Click "agregar al carrito"
 5. You will be redirected to the home page, and go to the shopping cart page.
 6. Click on "Ir a Pagar"
 7. You should see the view as in step 3
